### ISSUE TITLE
How does the functionality of this plugin differ from the official KMM plugin?

### ISSUE THREAD

**Issue by slipdef** [touchlab/xcode-kotlin#104]
I understand the major difference of course is that it's XCode plugin while KMM is Android plugin. But is it the major point? As for me it's irrelevant what IDE to choose for debugging MP code. Android studio feels more natural for debugging Kotlin code tho. Anyways I couldn't find in the description any word about KMM plugin. I'm at the point where I should choose between those two. So a few words about how xcode-kotlin plugin compares to KMM plugin would be much appreciated.

**Comment by kpgalligan** [comment]
> I understand the major difference of course is that it's XCode plugin while KMM is Android plugin. But is it the major point?

Yes. That is the major point.

> As for me it's irrelevant what IDE to choose for debugging MP code. Android studio feels more natural for debugging Kotlin code tho.

You should use Android Studio then. Other developers prefer Xcode, specifically iOS developers (mostly). Also, UI dev for iOS is not something Android Studio focuses on exactly. SwiftUI, etc. Most devs will prefer building the iOS UI in Xcode.

> So a few words about how xcode-kotlin plugin compares to KMM plugin would be much appreciated.

There's not much to compare. They both use LLDB to perform the actual debugging. This plugin makes no effort to provide a good editing experience for Kotlin. Just debugging, and source code coloring/highlighting.

TL;DR if you prefer Android Studio, you should use that. This plugin makes no effort to compete with the tools from JetBrains. It just provides support inside Xcode.
